src/verify.rs

```diff
@@ -65,6 +65,7 @@ fn compile_and_run_interactively(exercise: &Exercise) -> Result<bool, ()> {
         Err(output) => {
             warn!("Ran {} with errors", exercise);
             println!("{}", output.stdout);
+            println!("{}", output.stderr);
```

Comment:
I think this is a spurious change, isn't it?

Reply:
yeah, i think this should go in a separate PR if anything

Reply:
I think it's part of https://github.com/rust-lang/rustlings/pull/522/commits/9f61db5dbe38538cf06571fcdd5f806e7901e83a, which is already on `main`.

Reply:
@jrvidal @fmoko That's my bad, I think I based my change off of master, instead of 4.0.0 which had another commit in it.